metaflow/plugins/kfp/constants.py

```diff
@@ -3,9 +3,15 @@
 # Defaults for running MF on KFP
 # TODO: remove this when we're able to automatically package code and use it on KFP. We currently need the user supplied code URL (or this default URL) in order to download the code that is to be run on KFP
 DEFAULT_FLOW_CODE_URL = 'https://gist.githubusercontent.com/mon95/98d9d88571a0307a53b637d841295702/raw/df8bc1fa363f629271cfc1d34f3c2f7ca0b2e2cf/helloworld.py'
+DEFAULT_DOWNLOADED_FLOW_FILENAME = 'downloaded_flow.py'
+
 DEFAULT_KFP_YAML_OUTPUT_PATH = 'kfp_pipeline.yaml'
 DEFAULT_RUN_NAME = 'run_mf_on_kfp'
 DEFAULT_EXPERIMENT_NAME = 'mf-on-kfp-experiments'
 
 # TODO: This should (probably) be moved outside
-RUN_LINK_PREFIX = "https://kubeflow.corp.zillow-analytics-dev.zg-int.net/pipeline/#/runs/details/"
\ No newline at end of file
+RUN_LINK_PREFIX = "https://kubeflow.corp.zillow-analytics-dev.zg-int.net/pipeline/#/runs/details/"
+
+# TODO: Switch out of using this role
+S3_BUCKET = "s3://workspace-zillow-analytics-stage/aip/metaflow"
+S3_AWS_ARN = "arn:aws:iam::170606514770:role/dev-zestimate-role"
```

Comment:
let's make Zillow internal variables picked up from env variables.

Reply:
These variables are getting [read as environment variables](https://github.com/zillow/metaflow/blob/branch-and-join/metaflow/plugins/kfp/kfp.py#L110) inside the container op. I currently need a way to provide these values in some way to construct the pipeline and I've chosen to read them off of here while I construct the pipeline